src/types/Firestore.ts

```diff
@@ -19,6 +19,10 @@ export interface Experience {
   tables: Record<string, Record<string, Table>>;
 }
 
+export interface Settings {
+  showChat?: boolean;
+}
```

Comment:
One more for https://github.com/sparkletown/internal-sparkle-issues/issues/1170 ?

2 issues for the price of 1 PR

Reply:
I'd mark it as OOS